upgrade/src/migrateToMCD.js

```diff
@@ -0,0 +1,53 @@
+require('dotenv').config()
+const Web3 = require('web3')
+const multiSigWalletAbi = require('../abi/multiSigwallet')
+const proxyAbi = require('../../build/contracts/EternalStorageProxy').abi
+const foreignBridgeAbi = require('../../build/contracts/ForeignBridgeErcToNative').abi
+const confirmTransaction = require('./confirmTransaction')
+
+const {
+  FOREIGN_PRIVKEY,
+  FOREIGN_RPC_URL,
+  FOREING_BRIDGE_ADDRESS,
+  ROLE,
+  FOREIGN_START_BLOCK,
+  FOREIGN_GAS_PRICE
+} = process.env
+
+const web3 = new Web3(new Web3.providers.HttpProvider(FOREIGN_RPC_URL))
+const { address } = web3.eth.accounts.wallet.add(FOREIGN_PRIVKEY)
```

Comment:
May we output the address of this account and check that its balance is not zero?

Reply:
Added the file `upgrade/src/validatorState.js` that log the address of the validator and also check if it is not zero